### ISSUE TITLE
Add label to 'Other' field

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#891]
The "Other" field on our giving pages does not have a label attached to it. 
<img width="721" alt="Screenshot 2023-02-10 105323" src="https://user-images.githubusercontent.com/8452264/218149837-6548f12f-b3f1-4f2c-8f8a-6430633dc698.png">

**Comment by trubach** [comment]
Closing - The giving pages are deprecated since the GiveCampus rollout
